src/App.tsx

```diff
@@ -1,61 +1,142 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoWithUser } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+export const firstTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(({ id }) => id === todo.userId) || null,
+}));
 
 export const App = () => {
+  const [todoTitle, setTodoTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [userSelectedId, setUserSelectedId] = useState(0);
+  const [hasUserSelectedId, setHasUserSelectedId] = useState(false);
+  const [preparedTodos, setPreparedTodos] = useState(firstTodos);
+
+  function getUserById(userId: number) {
+    return usersFromServer.find(({ id }) => id === userId) || null;
+  }
+
+  function onSubmit(todo: TodoWithUser) {
+    setPreparedTodos(currentTodo => [...currentTodo, todo]);
+  }
+
+  function formReset() {
```

Comment:
Function names must begin with a verb:
```suggestion
  function resetForm() {
```